### PR TITLE
(googlechromebeta) - bug fix

### PR DESCRIPTION
**Pull request by nsleigh**
## Description
Better testing of the auto-update code so the code will now handle more scenarios.
Update package to v73.0.3683.67

## Motivation and Context
Bug fix

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [x] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](add_link_to_original_repository_location)
- [Open Issues](link_to_the_generic_location_of_open_issues) *Add the different issues underneath, and tick those that are fixed in this PR*
  - [ ] Issue 1 link
  - [ ] Issue 2 Link
- [ ] *Include the link to the opened PR that removes the package from the original location*
- [ ] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

**Comment by nsleigh** [comment]
It is unclear why the checks failed, is it possible to re-run them?

File: automatic/googlechromebeta/googlechromebeta.nuspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>googlechromebeta</id>
-    <version>73.0.3683.56-beta</version>
+    <version>73.0.3683.67-beta</version>
```
Comment (AdmiringWorm) [inline automatic/googlechromebeta/googlechromebeta.nuspec]: If this is the latest version available of the google chrome beta, the package won't ever be updated.

Please change the version number to something lower (or change it to `0.0`).

**Comment by AdmiringWorm** [comment]
@nsleigh the failure is caused by a bug in the testing script I've implemented.
It have problems when it comes to pre-release versions.

**Comment by nsleigh** [comment]
@AdmiringWorm Has this been accepted? Trying to understand what the messages here mean https://chocolatey.org/packages/GoogleChromeBeta.

**Comment by AdmiringWorm** [comment]
@nsleigh I had to reject the "stable" release that was pushed earlier. Otherwise we won't be able to push any new release at all.

The next time the appveyor update run is running, the beta version will be pushed.

**Comment by nsleigh** [comment]
@AdmiringWorm thanks, apologies for all the questions just trying make sure I am not missing anything in the process.

**Comment by nsleigh** [comment]
@AdmiringWorm my chocolatey username is nsleigh.

**Comment by AdmiringWorm** [comment]
I just added you now. You should have received an email from chocolatey with a link you'll need to confirm to be added.

**Comment by nsleigh** [comment]
Thanks - do you know why it is now saying "googlechromebeta has failed automated validation" it only seemed to say that after I was added as a maintainer.

**Comment by AdmiringWorm** [comment]
That is the review log.
Because there is actually a requirement of the package not being named anything related to a version, it is actually failing the validator.

However, due to it being pushed as a pre-release version it is still available for download when using the `--pre` argument.

**Pull request merged**